docs/integrations/sources/hubspot-migrations.md

```diff
@@ -1,5 +1,45 @@
 # HubSpot Migration Guide
 
+## Upgrading to 4.0.0
+
+:::note
+This change is only breaking if you are syncing streams `Deals Property History` or `Companies Peoperty History`.
```

Comment:
what about contacts?

Reply:
Contacts are not included in this update (stream was not mentioned in Oncall